test/test.js

```diff
@@ -9,11 +9,21 @@ var SAMPLE_JSON = {
   },
 };
 
-var SAMPLE_JSON_TWO = {
-  'JMON_TWO': {
-    'name': 'JMON_TWO',
-    'author': 'Bibek_TWO',
-    'url': 'https://github.com/bibekluitel/JMON_TWO',
+// var SAMPLE_JSON_TWO = {
```

Comment:
Let's remove this, if its not getting used.
We would want to keep the code clean. Let's now comment huge amount of code, removing is better option